paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -387,13 +494,14 @@ def get_autoscaling_info(marathon_client, service, instance, cluster, soa_dir):
                                           marathon_tasks=list(marathon_tasks.values()),
                                           mesos_tasks=mesos_tasks)
             error = get_error_from_utilization(utilization=utilization,
-                                               setpoint=autoscaling_params.pop('setpoint'),
+                                               setpoint=autoscaling_params['setpoint'],
                                                current_instances=service_config.get_instances())
             new_instance_count = get_new_instance_count(utilization=utilization,
                                                         error=error,
                                                         autoscaling_params=autoscaling_params,
                                                         current_instances=service_config.get_instances(),
-                                                        marathon_service_config=service_config)
+                                                        marathon_service_config=service_config,
+                                                        num_healthy_instances=len(marathon_tasks))
```

Comment:
Is len(marathon_tasks) really is the number of healthy instances or is it the number of instances that we have planned to run (without any guarantee that they are all healthy) ?